code/modules/mob/living/blood.dm

```diff
@@ -103,7 +103,7 @@
 			for(var/w in temp.wounds)
 				var/datum/wound/W = w
 				if(W.bleeding())
-					blood_max += (W.damage / 40)
```

Comment:
Whats the context on 40 & 60 here?
feels like a magic number right now (which we want to avoid)

Reply:
W.damage is the damage of the wound, so if you have 40 damage on your internal bleeding wound, it adds 1 blood loss to blood_max, 60 is a higher threshold and 50% higher